### DISCUSSION TITLE
How to filter out a specific entity using Audit.MongoClient

### DISCUSSION THREAD

**Discussion by feifles** [thepirat000/Audit.NET#675] [Q&A]
Hello!

I just started using this library and it's amazing, thank you for everyone involved in maintaning this repository.

My project uses MongoDb, I have successfully integrated audit logging but I do have a problem.

One of my entities is meant to have high volume, around 5Mi monthly.

I do not need to log the audit events for this entity, so I'm trying to make it said entity to be ignored.

I've looked around the doc and tried a few options but none seem to have access to the entity name that is being audit with Audit.MongoClient.

Is this implemented? How can I filter out a certain entity?

Thank you in advance!

**Comment by thepirat000** [comment]
As far as I know, MongoDB does not have a concept of 'entity.' It stores and retrieves BSON documents. I guess the only way to identify your C# entity type is by including the type name within the document itself.
